src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));
```

Comment:
we dont need to create Random and ColorSupplier instance each time when we call this method
also, `100` is a magic number